docs/community-tools.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: community tools
```

Comment:
```suggestion
title: Community tools
```

Reply:
shouldn't it be consistent with the rest of the docsite?

Reply:
Ahh roger that